ta/pkcs11/src/processing_symm.c

```diff
@@ -336,9 +409,35 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 
 	/*
 	 * Feed active operation with data
-	 * (PKCS11_FUNC_STEP_UPDATE/_ONESHOT)
```

Comment:
Why removing this comment here but not later when finalizing?

Reply:
For _ONESHOT Operation, for HMAC and AES cases, no active operation is performed.  It's only the update process where active operation is fed with data. Hence, we removed the comment from here and kept it as just "Feed active operation with data"

Reply:
That's fine, thanks for explaining it.